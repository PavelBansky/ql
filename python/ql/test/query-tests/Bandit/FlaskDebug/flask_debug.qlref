Bandit/flask_debug.ql