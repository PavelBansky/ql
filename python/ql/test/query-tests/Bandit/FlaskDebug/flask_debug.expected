| Test.py:10:1:10:19 | Attribute() |  A Flask app appears to be run with debug=True |
